tests/ServeCommandTest.php

```diff
@@ -64,4 +64,100 @@ public function testServeCommandExecuteWithDocRoot(): void
             $output
         );
     }
+
+    public function testServeCommandExecuteErrorIsAddressTaken(): void
+    {
+        $application = $this->container->get(Application::class);
+
+        $command = $application->find('serve');
+
+        $commandCreate = new CommandTester($command);
+
+        $commandCreate->setInputs(['yes']);
+
+        if (PHP_OS_FAMILY === 'Windows') {
+            $commandCreate->execute([
+                'address' => '127.0.0.1:445',
```

Comment:
What's on 445 port?

Reply:
It is a port used by windows, and as it is busy I can cause the error in the adresss.